_data/faq.yml

```diff
@@ -458,46 +458,59 @@
 
       - id: data-on-sd
         en:
-            title: c:geo data is always stored in internal memory!
+            title: How can I store c:geo data on my SD card?
             content: |
                 There are two things we are storing on your device:
```

Comment:
Actually we have more than two things we store. Do we want to mention backup, field-notes, gpx im-/export, spoiler images and log photos?